src/mpyl/steps/build/dockerbuild.py

```diff
@@ -71,32 +71,22 @@ def execute(self, step_input: Input) -> Output:
             # log in to registry, because we may need to pull in a base image
             login(logger=self._logger, registry_config=docker_registry_config)
 
-        with open(".dockerignore", "w+", encoding="utf-8") as ignore_file:
-            contents = "\n".join(DOCKER_IGNORE_DEFAULT)
-            ignore_file.write(contents)
-
-        build_args: dict[str, str] = (
-            {
-                arg.key: arg.get_value(step_input.run_properties.target)
-                for arg in step_input.project.build.args.plain
-            }
-            if step_input.project.build
-            else {}
-        )
-
         success = build(
             logger=self._logger,
             root_path=docker_config.root_folder,
             file_path=dockerfile,
             image_tag=image_tag,
             target=build_target,
             registry_config=docker_registry_config,
-            build_args=build_args,
         )
         artifact = input_to_artifact(
             ArtifactType.DOCKER_IMAGE, step_input, spec=DockerImageSpec(image=image_tag)
         )
 
+        with open(".dockerignore", "w+", encoding="utf-8") as ignore_file:
+            contents = "\n".join(DOCKER_IGNORE_DEFAULT)
+            ignore_file.write(contents)
+
```

Comment:
@SamTheisens why was the dockerignore part after the build?

Reply:
Yeah, good question. IIRC it was necessary to avoid the docker layers getting invalidated every build. But I think it's a good idea to validate that assumption.